blazar_dashboard/static/leases/js/calendar/lease_chart.js

```diff
@@ -246,6 +250,13 @@
             }
           ]
         },
+        // Free performance
+        animations: {
+          enabled: false
+        },
+        dataLabels: {
```

Comment:
Does this change the way the calendar looks at all?

Reply:
It doesn't look any different to me
![image](https://user-images.githubusercontent.com/14908880/173697950-37d55b7a-a2ac-491a-b211-11ac7e74dad0.png)

Reply:
@super-cooper Did you check how it looks with active allocations in the calendar? I wonder if they are a source of lag. You could just reserve all of them at once.

Reply:
![image](https://user-images.githubusercontent.com/14908880/173917545-0ae6ccea-c804-4fae-a8ac-34a68b12e623.png)
looks good to me :)